### ISSUE TITLE
Install MXNet on Ubuntu 18 and Julia-1.0.3?

### ISSUE THREAD

**Issue by SamuelLarkin** [apache/incubator-mxnet#13836]
Hi,
  disclaimer ;) I'm a complete newbie in Julia.
I want to install MXNet for Julia-1.0.3 under Ubuntu18 but I get
```
(v1.0) pkg> add MXNet
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package MXNet [a7949054]:
 MXNet [a7949054] log:
 ├─possible versions are: [0.0.1-0.0.8, 0.1.0, 0.2.0-0.2.3, 0.3.0] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.0.1-0.0.8, 0.1.0, 0.2.0-0.2.3, 0.3.0]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```
which leads me to believe that MXNet-1.3.1 is not available for Julia-1.0.3.

Am I doing something wrong here?

**Comment by marcoabreu** [comment]
Hey Samuel,

welcome to MXNet! We currently only support Julia 0.6, but are working on 0.7 and 1.0 support here: #12845

CC @iblis17

**Comment by SamuelLarkin** [comment]
Thanks for the info.  I guess I'm too eager.  Since I'm new to julia, I thought that starting with the latest and greatest version of julia would be best but that doesn't take into account that other projects have to catch up.  I'm fine with giving julia-0.6 a try.

Best of luck with your release of MXNet for Julia.

FYI I'm a huge fan of Sockeye which is build on top of MXNet.

**Comment by lanking520** [comment]
@SamuelLarkin Close this issue for now. Please feel free to reopen it if you are facing more problems with it.

**Comment by aaronmarkham** [comment]
> Hi,
> You can try out that PR via following instructions:
> 
> ```
> git clone https://github.com/apache/incubator-mxnet.git
> cd incubator-mxnet
> git fetch origin pull/12845/head:julia-1.0
> git checkout julia-1.0
> ```
> 
> Then, start your Julia shell
> 
> ```julia
> using Pkg
> Pkg.develop(PackageSpec(name = "MXNet", path = "./julia"))
> Pkg.build("MXNet")
> ...
> using MXNet
> ```

Anyone else troubleshooting setting up Julia with MXNet should note that this PR has been merged and MXNet now supports 0.7 and 1.0.x.

**Comment by lewisl** [comment]
Well, no--it doesn't seem to work at all:

```
(v1.1) pkg> add MXNet
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package MXNet [a7949054]:
 MXNet [a7949054] log:
 ├─possible versions are: [0.0.1-0.0.8, 0.1.0, 0.2.0-0.2.3, 0.3.0] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.0.1-0.0.8, 0.1.0, 0.2.0-0.2.3, 0.3.0]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```
